moler/device/abstract_device.py

```diff
@@ -43,6 +46,27 @@ def name(self, value):
         :return: None
         """
 
+    @property
+    @abc.abstractmethod
+    def external_name(self):
```

Comment:
Better naming would be `public_name` - see comment below

Moreover, `external` to what?

Reply:
It was alias_name but change to external_name  after Friday's call ;)

Reply:
Done